css/style.css

```diff
@@ -124,14 +181,14 @@ h2 {
 .rightPartSection {
     border-radius: 25px;
     border: 5px solid rgb(25, 0, 187);
-    display: flow-root;
-    width: 49%;
+    /*display: flow-root;*/
+    /*width: 49%;
     height: 100%;
     padding: 0%;
-    margin: 0%;
+    margin: 0%;*/
     background-color: #000000;
-    align-items: center;
-    align-content: center;
+    /*align-items: center;
+    align-content: center;*/
 }

```

Comment:
Value commented I think, I don't know if it is normal.

Reply:
it is